tests/e2e/setup.go

```diff
@@ -94,40 +92,37 @@ func (suite *CCVTestSuite) SetupTest() {
 	suite.providerChain.NextBlock()
 
 	// initialize the consumer chain with the genesis state stored on the provider
-	consumerGenesis, found := providerKeeper.GetConsumerGenesis(
+	consumerGenesisState, found := providerKeeper.GetConsumerGenesis(
 		suite.providerCtx(),
 		suite.consumerChain.ChainID,
 	)
 	suite.Require().True(found, "consumer genesis not found")
-	consumerKeeper.InitGenesis(suite.consumerCtx(), &consumerGenesis)
-	suite.providerClient = consumerGenesis.ProviderClientState
-	suite.providerConsState = consumerGenesis.ProviderConsensusState
+	consumerKeeper.InitGenesis(suite.consumerCtx(), &consumerGenesisState)
+
+	// Confirm client and cons state for consumer were set correctly in InitGenesis
+	consumerEndpointClientState, consumerEndpointConsState := suite.GetConsumerEndpointClientAndConsState()
+	suite.Require().Equal(consumerGenesisState.ProviderClientState, consumerEndpointClientState)
+	suite.Require().Equal(consumerGenesisState.ProviderConsensusState, consumerEndpointConsState)
 
 	// create path for the CCV channel
 	suite.path = ibctesting.NewPath(suite.consumerChain, suite.providerChain)
 
-	// update CCV path with correct info
-	// - set provider endpoint's clientID
-	consumerClient, found := providerKeeper.GetConsumerClientId(
+	// Set provider endpoint's clientID
+	providerEndpointClientID, found := providerKeeper.GetConsumerClientId(
 		suite.providerCtx(),
 		suite.consumerChain.ChainID,
 	)
+	suite.Require().True(found, "provider endpoint clientID not found")
+	suite.path.EndpointB.ClientID = providerEndpointClientID
+
+	// Set consumer endpoint's clientID
+	consumerEndpointClientID, found := consumerKeeper.GetProviderClientID(suite.consumerChain.GetContext())
+	suite.Require().True(found, "consumer endpoint clientID not found")
+	suite.path.EndpointA.ClientID = consumerEndpointClientID
+
+	// Confirm client config is now correct
+	suite.TestEndpointsClientConfig()
 
-	suite.Require().True(found, "consumer client not found")
-	suite.path.EndpointB.ClientID = consumerClient
-	// - set consumer endpoint's clientID
-	providerClient, found := consumerKeeper.GetProviderClientID(suite.consumerChain.GetContext())
-	suite.Require().True(found, "provider client not found")
-	suite.path.EndpointA.ClientID = providerClient
-	// - client config
-
-	trustingPeriodFraction := suite.providerApp.GetProviderKeeper().GetTrustingPeriodFraction(suite.providerCtx())
-	providerUnbondingPeriod := suite.providerApp.GetStakingKeeper().UnbondingTime(suite.providerCtx())
-	suite.path.EndpointB.ClientConfig.(*ibctesting.TendermintConfig).UnbondingPeriod = providerUnbondingPeriod
```

Comment:
These setters were removed because we don't use any functionality from the IBC testing package that require the endpoint client configs to be set.